report/src/problem_analysis/understanding.tex

```diff
@@ -0,0 +1,31 @@
+To establish knowledge about the domain and issues with the current system for managing meeting arrangements, two interviews with the stakeholders was conducted.  
+The goals of these interviews were twofold: 
+First and foremost, we wanted  to understand the current issues the stakeholders are facing. 
+To figure this out we conducted an unstructured interview\cite{robson2002real}.
```

Comment:
Should we say why we chose an unstructured interview?

i.e. to optimize for unexpected discoveries